model/application_authentication.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"strconv"
+	"time"
+)
+
+type ApplicationAuthentication struct {
+	ID        int64     `gorm:"primarykey" json:"id"`
+	CreatedAt time.Time `json:"created_at"`
+	UpdatedAt time.Time `json:"updated_at"`
+
+	TenantID int64
+	Tenant   Tenant
+
+	ApplicationID    int64 `json:"application_id"`
+	AuthenticationID int64 `json:"authentication_id"`
```

Comment:
In order for relations to work - we need a few lines added:
```suggestion
	ApplicationID    int64 `json:"application_id"`
	Application      Application
	AuthenticationID int64 `json:"authentication_id"`
	Authentication   Authentication
```

Reply:
Fixed, thanks!